python-avd/pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[build-system]
+requires = ["setuptools>=61.2", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name="pyavd"
+version = "4.0.0-dev9"
+authors=[{ name = "Arista Networks", email = "ansible@arista.com"}]
+description="Arista validated designs"
+readme = "README.md"
+license = { file = "LICENSE" }
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Development Status :: 5 - Production/Stable",
```

Comment:
Not production. Still under development / alpha stage

Reply:
Development Status :: 2 - Pre-Alpha

Reply:
Fixed!